src/wallet/wallet.cpp

```diff
@@ -4581,3 +4581,31 @@ ScriptPubKeyMan* CWallet::AddWalletDescriptor(WalletDescriptor& desc, const Flat
 
     return ret;
 }
+
+void CWallet::LoadWalletID(const uint160& id)
```

Comment:
Is there a reason to limit it to 160-bit? Seems like to future-proof, 256-bit ids would be better. And even if we end up only generating 160-bit ids now, it would be nice to load longer ones so we don't have to break compatibility if we ever generate more...

Reply:
The current BDB ID is 160 bits long. I don't think limiting to "just" 160 bits will be a problem. Why would we need to load longer ones?

Reply:
Probably 160-bit is enough, but I can't predict the future.